test/test_layer_pot.py

```diff
@@ -457,25 +456,32 @@ def test_3d_jump_relations(actx_factory, relation, visualize=False):
 
     qbx_order = target_order
 
+    if relation == "sp":
+        resolutions = [10, 14, 18]
+    else:
+        resolutions = [6, 10, 14]
+
     from pytools.convergence import EOCRecorder
     eoc_rec = EOCRecorder()
 
-    for nel_factor in [6, 10, 14]:
+    for nel_factor in resolutions:
         from meshmode.mesh.generation import generate_torus
         mesh = generate_torus(
-                5, 2, order=target_order,
-                n_major=2*nel_factor, n_minor=nel_factor)
+                5, 2, n_major=2*nel_factor, n_minor=nel_factor,
+                order=target_order,
+                )
 
         from meshmode.discretization import Discretization
         from meshmode.discretization.poly_element import \
             InterpolatoryQuadratureSimplexGroupFactory
-        pre_discr = Discretization(
+        pre_density_discr = Discretization(
                 actx, mesh,
-                InterpolatoryQuadratureSimplexGroupFactory(3))
+                InterpolatoryQuadratureSimplexGroupFactory(target_order))
```

Comment:
I'm actually not quite sure how this was all converging before if the discretization always used 3rd order here, even if the `target_order = 4`. Hm..

Reply:
Well, it would have had a hard time converging better than third order... :)